From effcaa0a3df6e04c672803cf71201ec63ca31e0e Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.ledkov@surgut.co.uk>
Date: Tue, 18 Mar 2025 15:40:44 +0000
Subject: [PATCH] database: initiate apk_flags to NO_CACHE by default

This ensures broken cache, missing cache, corrupted cache does not
impact apk operations.

Also it ensures after the operation is completed, the caches are clean
(never created) keeping individual container layers small.
---
 src/database.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/database.c b/src/database.c
index dc5e4fd..13b6fb0 100644
--- a/src/database.c
+++ b/src/database.c
@@ -47,7 +47,7 @@ enum {
 };
 
 int apk_verbosity = 1;
-unsigned int apk_flags = 0, apk_force = 0;
+unsigned int apk_flags = APK_NO_CACHE, apk_force = 0;
 
 static apk_blob_t tmpprefix = { .len=8, .ptr = ".apknew." };
 
-- 
2.43.0

